File: 2624-difference-between-element-sum-and-digit-sum-of-an-array/solution.c
int differenceOfSum(int* nums, int numsSize) {
    int sum1=0;
    int sum2=0;
    for(int i=0;i<numsSize;i++){
        sum1+=nums[i];
        while(nums[i]!=0){
            int rem=nums[i]%10;
            sum2+=rem;
            nums[i]/=10;
        }
    }
    int ans=abs(sum1-sum2);
    return ans;
}

File: 2624-difference-between-element-sum-and-digit-sum-of-an-array/solution.java
class Solution {
    private int sum(int[] arr){
        int sum = 0;
        int sum2 = 0;
        for(int i: arr) {
            sum += i;
            while(i!=0){
                sum2 += (i%10);
                i /= 10;
            }
        }
        return Math.abs(sum -sum2);
    }

    public int differenceOfSum(int[] nums) {
        return sum(nums);
    }
}
